packages/twenty-docs/src/components/token-form.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import { parseJson } from "nx/src/utils/json";
+import tokenForm from '!css-loader!./token-form.css';
+import { TbLoader2 } from "react-icons/tb";
+
+export type TokenFormProps = {
```

Comment:
we are missing a linter on these files, it would be great to add one in the future (but as we plan to migrate to nextjs, it doesn't worth it I think)